js/public/app.js

```diff
@@ -1792,8 +1792,8 @@ app.directive('ocdatetimepicker', ["$compile", "$timeout", function($compile, $t
 			disabletime: '=disabletime'
 		},
 		link: function (scope, element, attrs, ngModelCtrl) {
-			var templateHTML = '<input type="text" ng-model="date" class="events--date" />';
-			templateHTML += '<input type="text" ng-model="time" class="events--time" ng-disabled="disabletime"/>';
+			var templateHTML = '<input type="text" ng-model="date" class="events--date" tabindex="104" />';
```

Comment:
this probably won't work because this html code can appear more than once in the sidebar editor.
Let me prepare a pr to set the tab index with variables.